api/src/test/java/org/pmiops/workbench/db/dao/InstitutionDaoTest.java

```diff
@@ -3,65 +3,192 @@
 import static com.google.common.truth.Truth.assertThat;
 import static com.google.common.truth.Truth8.assertThat;
 
-import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Sets;
+import java.util.Set;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.pmiops.workbench.db.model.DbInstitution;
+import org.pmiops.workbench.db.model.DbInstitutionEmailAddress;
+import org.pmiops.workbench.db.model.DbInstitutionEmailDomain;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
 import org.springframework.dao.DataIntegrityViolationException;
 import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.annotation.DirtiesContext.ClassMode;
 import org.springframework.test.context.junit4.SpringRunner;
 
 @RunWith(SpringRunner.class)
 @DataJpaTest
-@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+@DirtiesContext(classMode = ClassMode.BEFORE_EACH_TEST_METHOD)
 public class InstitutionDaoTest {
+
   @Autowired InstitutionDao institutionDao;
 
-  private DbInstitution testInst = new DbInstitution("Broad", "The Broad Institute");
+  private DbInstitution institutionNoEmailAddressesOrDomains;
```

Comment:
these are a mouthful and also very hard to distinguish (b/c the different letters are in the middle). Maybe `simpleInstitution` and `emailPopulatedInstition` or something?

Reply:
will update